lambdaclass/curse_of_myrra
# Display states Icon in the UI

With this we can display in the UI the states that affect the player . We currently have Poisoned, Paralyzed and Slowed.

How to test:

- Start a game with min 2 players
- Apply Slow,Poison or Paralyzed effect to one of the player
- The player affected should see a icon on the top of the screen representing the state applied

closes #661
